cpp/ps0000/ps0000soln.cpp

```diff
@@ -1,8 +1,9 @@
-#include <iostream>
+#include <iostream> // 2/2
 using namespace std;
 
-int main(void)
+int main(void) // 1/1
 {
-  cout << "Hello, World!" << "\n";
-  return 0;
-}
\ No newline at end of file
+  cout << "Hello, World!" << "\n"; // 4/4

```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [x] output is as required `Hello, World!` in a single line